.travis.yml

```diff
@@ -6,45 +6,43 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction --no-plugins"
+    - COMPOSER_ARGS="--no-interaction"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 5.6
+    - php: 7.3
       env:
         - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
+        - BENCHMARKS=true
+        - CS_CHECK=true
+        - TEST_COVERAGE=true
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.4
       env:
         - DEPS=latest
-        - BENCHMARKS=true
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
-    - php: 7.2
+    - php: nightly
       env:
         - DEPS=lowest
-    - php: 7.2
+        - COMPOSER_ARGS="$COMPOSER_ARGS --ignore-platform-reqs"
```

Comment:
Are we ok with `ignore-platform-reqs` ? I think it's risky to use something that is not right tested. Which dependencies require this flag?

Reply:
Since 3rd-party dependencies do not provide proper constraints, this is still needed.
It is risky but thats how it is for now and thats how it is in almost every other laminas/mezzio package due to the lack of support in other libs.